src/components/forms/AcademicGrantsForm.tsx

```diff
@@ -115,1388 +119,1400 @@ export const AcademicGrantsForm: FC = () => {
+                  value={value}
+                  fontSize='input'
+                  colorScheme='white'
+                  mt={4}
+                >
+                  <Stack direction='row'>
+                    <Radio
+                      id='canTheEFReachOut-yes'
+                      size='lg'
+                      name='canTheEFReachOut'
+                      value='Yes'
+                      defaultChecked
+                      mr={8}
+                    >
+                      <PageText fontSize='input'>Yes</PageText>
+                    </Radio>
+
+                    <Radio id='canTheEFReachOut-no' size='lg' name='canTheEFReachOut' value='No'>
+                      <PageText fontSize='input'>No</PageText>
+                    </Radio>
+                  </Stack>
+                </RadioGroup>
+              </FormControl>
+            )}
+          />
+
+          <FormControl id='additional-info-control' mb={12}>
+            <FormLabel htmlFor='additionalInfo' mb={1}>
+              <PageText fontSize='input'>
+                Do you have any questions about this grants round, or is there anything else
+                you&apos;d like to share?
+              </PageText>
+            </FormLabel>
+
+            <PageText as='small' fontSize='helpText' color='brand.helpText'>
+              Is there anything we didn&apos;t cover in the above questions? Feel free to add any
+              relevant links here. This is optional.
+            </PageText>
+
+            <Textarea
+              id='additionalInfo'
+              bg='white'
               borderRadius={0}
-              type='submit'
-              isDisabled={!isValid}
-              _hover={{ bg: 'brand.hover' }}
-              whileHover={{ x: -1.5, y: -1.5 }}
-              onMouseEnter={() => setButtonHovered(true)}
-              onMouseLeave={() => setButtonHovered(false)}
-              pointerEvents={!isValid ? 'none' : 'auto'}
-            >
-              <ImportantText as='h3' color='white'>
-                Submit Application
-              </ImportantText>
-
-              <Flex pl={5}>
-                <Image src={planeVectorSVG} alt='paper plane vector' height='29px' width='32px' />
-              </Flex>
-            </MotionButton>
-          </Box>
-        </Center>
-      </form>
+              borderColor='brand.border'
+              color='brand.paragraph'
+              fontSize='input'
+              h='150px'
+              mt={3}
+              {...register('additionalInfo', {
+                maxLength: 32768
+              })}
+            />
+
+            {errors?.additionalInfo?.type === 'maxLength' && (
+              <Box mt={1}>
+                <PageText as='small' fontSize='helpText' color='red.500'>
+                  Additional info cannot exceed 32768 characters.
+                </PageText>
+              </Box>
+            )}
+          </FormControl>
+
+          <Center mb={12}>
```

Comment:
The only significant change in the html

Reply:
Note: only changes in the forms html are the inclusion of `<FormProvider>` and `<Captcha />`. You can ignore the rest of the diff, nothing has changed.